firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -69,21 +69,21 @@ extension BrowserViewController: WKUIDelegate {
         initiatedByFrame frame: WKFrameInfo,
         completionHandler: @escaping () -> Void
     ) {
+        defer {
+            completionHandler()
+        }
```

Comment:
Yes true that makes sense, i did that because for MessageAlert the completionHandler doesn't require a value to be passed and so i could call right after the method returned. I should change it since it is not the intended behavior. Thanks for the review @mattreaganmozilla i'll dig into this more